### ISSUE TITLE
Following setup instructions, demo site not working.

### ISSUE THREAD

**Issue by COLABORATI** [stephenmcd/drum#36]
Well, I do not know if it makes sense to paste all the error messages, however it results in a
```
A server error occurred.  Please contact the administrator.
```
when visiting 127.0.0.1/admin or 127.0.0.1

the console output has this last line:
```
RuntimeError: Model class mezzanine.pages.models.Page doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS.
```

Also there are some warnings before that:
```
System check identified some issues:

WARNINGS:
?: (mezzanine.core.W01) Please update your settings to use the TEMPLATES setting rather than the deprecated individual TEMPLATE_ settings. The latter are unsupported and correct behaviour is not guaranteed. Here's a suggestion based on on your existing configuration:

TEMPLATES = [{u'APP_DIRS': True,
              u'BACKEND': u'django.template.backends.django.DjangoTemplates',
              u'DIRS': (u'/home/xxx/tmp/test/mezzanine-drum/drumtest/templates',),
              u'OPTIONS': {u'builtins': [u'mezzanine.template.loader_tags'],
                           u'context_processors': (u'django.contrib.auth.context_processors.auth',
                                                   u'django.contrib.messages.context_processors.messages',
                                                   u'django.core.context_processors.debug',
                                                   u'django.core.context_processors.i18n',
                                                   u'django.core.context_processors.static',
                                                   u'django.core.context_processors.media',
                                                   u'django.core.context_processors.request',
                                                   u'django.core.context_processors.tz',
                                                   u'mezzanine.conf.context_processors.settings',
                                                   u'mezzanine.pages.context_processors.page')}}]

?: (mezzanine.core.W02) TEMPLATE_DEBUG and DEBUG settings have different values, which may not be what you want. Mezzanine used to fix this for you, but doesn't any more. Update your settings.py to use the TEMPLATES setting to have template debugging controlled by the DEBUG setting.

System check identified 2 issues (0 silenced).

```

Is there a quick fix for this? I just wanted to take a look at mezzanine...

**Comment by COLABORATI** [comment]
After applying the obviously needed changes to settings.py it _seems_ to work, but I am not sure, no tests. At least the error is not there, I can login and it looks like a website. 

In the settings.py file in the repo there are some apps commented out that seem to be crucial for a working setup, don't know why:
```
INSTALLED_APPS = (
    "django.contrib.admin",
    "django.contrib.auth",
    "django.contrib.contenttypes",
    "django.contrib.redirects",
    "django.contrib.sessions",
    "django.contrib.sites",
    "django.contrib.sitemaps",
    "django.contrib.staticfiles",
    "drum.links",
    "mezzanine.boot",
    "mezzanine.conf",
    "mezzanine.core",
    "mezzanine.generic",
    "mezzanine.accounts",
    #"mezzanine.blog",
    #"mezzanine.forms",
    #"mezzanine.pages",
    #"mezzanine.galleries",
    #"mezzanine.twitter",
    #"mezzanine.mobile",
)

```

when I remove the hash and paste the recommended TEMPLATES settings it seems to work.

**Comment by COLABORATI** [comment]
#37

**Comment by stephenmcd** [comment]
Already fixed in d863e396bbbae0348d67ae739b0e0fd5284aa9f1
